toolbox/build/build.sh

```diff
@@ -8,6 +8,7 @@ OS_MIGRATE_DIR=$(realpath "$DIR/../..")
 ### PACKAGES ###
 
 dnf clean all
+dnf -y update
```

Comment:
Definetely is a good idea to update the container packages before anything else, thanks!